subprojects/packagefiles/libarchive/cpio/meson.build

```diff
@@ -0,0 +1,17 @@
+src = [
+  'cmdline.c',
+  'cpio.c',
+  ]
+
+if host_machine.system() == 'windows'
+  src += 'cpio_windows.c'
+endif
+
+bsdcpio = executable(
+  'bsdcpio',
+  files(src),
```

Comment:
If you're going to use files() on the basis that `muon fmt` can discern the order of literal arguments and lexicographically sort them, then that's one thing. Passing an array as a variable reference to files() is pointless though because it doesn't help `muon fmt` *and* it's unneeded / potentially less readable.

Reply:
Right, that one was dumb…